include/libtorrent/aux_/piece_picker.hpp

```diff
@@ -301,7 +301,6 @@ namespace libtorrent::aux {
 		// that are being downloaded
 		int add_blocks(piece_index_t piece, typed_bitfield<piece_index_t> const& pieces
 			, std::vector<piece_block>& interesting_blocks
-			, std::vector<piece_block>& backup_blocks
 			, std::vector<piece_block>& backup_blocks2
```

Comment:
This is just a stylistic issue, but here within the header you took out the backup_blocks parameter and kept backup_blocks2, however in the definition you reversed this.